src/main/java/com/mg105/entities/items/HealthPotion.java

```diff
@@ -0,0 +1,19 @@
+package com.mg105.entities.items;
+
+import com.mg105.entities.Item;
+import com.mg105.utils.ItemConstants;
+
+public class HealthPotion extends Item {
+
+    // This number could change later
+
+    public final static int HEALING_POINTS = 25;
```

Comment:
Nice class! After playtesting our game we can always change the value of healing points to provide a balanced game for the player.